src/ldclient_event_process_server.erl

```diff
@@ -152,6 +152,15 @@ terminate(_Reason, _State) ->
 code_change(_OldVsn, State, _Extra) ->
     {ok, State}.
 
+%% @doc Redact SDK key from state for logging
+%% @private
+%%
+%% @end
+format_status(#{state := State}) ->
+    #{state => State#{sdk_key => "[REDACTED]"}};
```

Comment:
`State#{sdk_key => "[REDACTED]"}` would be like `{...State, sdk_key: "[REDACTED]"]}` in JS.

Reply:
Alternatively, should we log the auth suffix? That might make it more useful?

Reply:
For erlang each SDK instance has a `tag` and that will be included in the state. So `default` for the default instance, and then subsequent instances you have to provide a `tag` for.

Reply:
Which provides similar functionality to the SDK key suffix, without any potential issues from short keys, newlines, or other malformed nonsense.